REFERENCE.md

```diff
@@ -660,6 +726,62 @@ The hash on which to operate
 
 ## Data types
 
+### Yum::RpmArch
+
+Possible rpm architectures
+See https://github.com/rpm-software-management/rpm/blob/master/rpmrc.in
+example noarch
+example x86_64
+
+Alias of `Enum['noarch', 'x86_64', 'i386', 'arm', 'ppc64', 'ppc64le', 'sparc64', 'ia64', 'alpha', 'ip', 'm68k', 'mips', 'mipsel', 'mk68k', 'mint', 'ppc', 'rs6000', 's390', 's390x', 'sh', 'sparc', 'xtensa']`
+
+### Yum::RpmEpoch
+
+valid epoch of rpm, it is always and integer greater than 0
+example 0
+example 1
+
+Alias of `Integer[0]`
+
+### Yum::RpmName
+
+Valid RPM names can be almost anything
+
+#### Examples
+
+##### openssh
+
+```puppet
+
```

Comment:
Missing?

Reply:
Bit lost about this one. Maybe a rendering one.